paper_demo/test_client.py

```diff
@@ -22,7 +22,7 @@ class GladierTestClient(GladierBaseClient):
             "from_storage_transfer_source_path": "/PTYCHO/fly001",
             
             # Add your Globus Collection here
-            # "from_storage_transfer_destination_endpoint_id": "", 
+            "from_storage_transfer_destination_endpoint_id": "", 
```

Comment:
I think we want to include the comment here. If it's commented out, Gladier will raise an error reminding people to set it.

Reply:
Oh I see. I just figured if they were editing the document they'd see the two blank spaces and the comment may be confusing. If a nicer error is popped up then I agree.